crates/bdk/src/wallet/tx_builder.rs

```diff
@@ -94,8 +94,8 @@ impl TxBuilderContext for BumpFee {}
 ///     let mut builder = wallet.build_tx();
 ///     builder
 ///         .ordering(TxOrdering::Untouched)
-///         .add_recipient(addr1.script_pubkey(), 50_000)
-///         .add_recipient(addr2.script_pubkey(), 50_000);
+///         .add_recipient(addr1.script_pubkey(), bitcoin::Amount::from_sat(50_000))
+///         .add_recipient(addr2.script_pubkey(), bitcoin::Amount::from_sat(50_000));
```

Comment:
You are importing in L50

```suggestion
///         .add_recipient(addr1.script_pubkey(), Amount::from_sat(50_000))
///         .add_recipient(addr2.script_pubkey(), Amount::from_sat(50_000));
```